pages/common/npm-query.md

```diff
@@ -0,0 +1,32 @@
+# npm query
+
+> Print an array of dependency objects using CSS-like selectors.
+> More information: <https://docs.npmjs.com/cli/v8/commands/npm-query>.
+
+- Print direct dependencies:
+
+`npm query ':root > *'`
+
+- Print all direct production/development dependencies:
+
+`npm query ':root > .{{prod|dev}}'`
+
+- Print dependencies with a specific name:
+
+`npm query '#{{package_name}}'`
+
+- Print dependencies with a specific name and within a semantic versioning range:
+
+`npm query #{{package_name}}@{{semantic_version}}`
+
+- Print dependencies with no dependencies:
+
+`npm query ':empty'`
+
+- Find all dependencies with postinstall scripts and uninstall them:
+
+`npm query ":attr(scripts, [postinstall])" | jq 'map(.name) | join("\n")' -r | xargs -I {} npm uninstall {}`
```

Comment:
Is this a little too specific?

It seems to be more of a tutorial than a basic usage of a command.

Reply:
Hmm… I think tldr usually consists of **common** use-cases, not **basic** use-cases.
With that in mind, if this is considered a common use-case, I don't see why not.

Reply:
I think the `npm`'s man page has this use case because the controversial of scripts with `postinstall` and the associated security risk.